app/models/protocol.rb

```diff
@@ -230,8 +230,8 @@ def unique_rm_id_to_protocol
     when "Short/Long Title"
       where(title_query.join(' OR ')).distinct
     when ""
-      all_query = [authorized_user_query, hr_query, pi_query, protocol_id_query, pro_num_query, rmid_query, title_query]
-      joins(:identities).joins(:human_subjects_info).
+      all_query = [authorized_user_query, pi_query, protocol_id_query, title_query]
```

Comment:
Wasn't this a requested feature, though? Can we look into whether there's another way we can still use these attributes?